addons/microsoft_calendar/models/calendar.py

```diff
@@ -111,6 +112,9 @@ def _check_organizer_validation(self, sender_user, partner_included):
                 raise ValidationError(
                     _("It is necessary adding the proposed organizer as attendee before saving the event."))
 
+    def _check_organizer_validation_conditions(self):
```

Comment:
Careful that this is `microsoft_calendar`, appointment doesn't depend on this module so you should add the method in base calendar.

Also, your override cannot take in more/different arguments than the original method, as in regular inheritance. Here I would just pass in "vals" and let the method sort out what it needs.

Reply:
Thanks for checking! Oversight on my part. I wrote this declaration before writing the appointment side and realizing I would need something from vals.